spec/support/i18n.rb

```diff
@@ -1,3 +1,14 @@
 RSpec.configure do |config|
   config.include AbstractController::Translation
+
+  def with_translations(locale, translations)
+    original_backend = I18n.backend
+
+    I18n.backend = I18n::Backend::KeyValue.new Hash.new, true
```

Comment:
I'm curious why you used `Hash.new` instead of `{}`? I don't see `Hash.new` used very often